src/lib/components/compositions/checkbox/animated.story.svelte

```diff
@@ -6,32 +6,34 @@
   import Form from '$components/primitives/form/form.svelte';
 
   export let checkValidity: ((event: Event) => boolean) | null = null;
-  export let errorMessage = '';
+  export let message = '';
```

Comment:
Changing from `errorMessage` to just generic `message` across all components as we might want to communicate non-error information to the user at the field level.